### DISCUSSION TITLE
Question about plotting in simulating invasion

### DISCUSSION THREAD

**Discussion by likeduck** [PMEAL/OpenPNM#2696] [Q&A]
Dear prof，

I have learned the simulating invasion  in openpnm'tutorial, the difference is that i used my PNM. Its information is: 

![捕获](https://user-images.githubusercontent.com/117707506/219382290-c26f00c0-a121-4d99-bf23-5a1cca00f460.PNG)

and visualization is: 

![Figure_1](https://user-images.githubusercontent.com/117707506/219383876-65ca8f45-ec2d-431a-bed1-c75fda72f184.png)

When i run the plotting code in the end, i got this: 

![Figure_2](https://user-images.githubusercontent.com/117707506/219384202-bb7457f6-5467-411d-ac96-49a4b7ace64a.png)

I am sure it is realistic, but not satisfactory, I want to get a more ' beautiful ' curve, but do not know how to make it more ' smooth '.
Could you please give me some advice?

**Comment by jgostick** [comment]
I don't know what's going on this with one, but the green line is wrong.

**Comment by likeduck** [comment]
Probably because large voxels of PNM，I try more times using other PNM. 

Thanks!
